docs/source/guide/dataset_create.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Create a Dataset
+short: Create a Dataset
+date: 2023-08-16 11:52:38
+tier: enterprise
+order: 0
+order_enterprise: 405
+meta_title: Create a Dataset to use with data discovery in Label Studio Enterprise
+meta_description: How to create a Dataset in Label Studio Enterprise using Google Cloud, Azure, or AWS.
+hide_sidebar: true
+---
+
+!!! note
+    At this time, we only support building datasets from unstructured data, meaning that the data must be in individual files rather than a structured format such as CSV or JSON.
+
+!!! note
+    To create a new Dataset, your [user role](manage_users#Roles-in-Label-Studio-Enterprise) must have Owner or Administrator permissions. 
+
+## Before you begin
+
+Datasets are retrieved from your cloud storage environment. As such, you will need to provide the appropriate access key to pull data from your cloud environment.
+
+
+## Datasets using Google Cloud Storage
```

Comment:
I think it likely makes sense to have redundant docs in this case (since Datasets works pretty differently from Projects in some cases), but just to be sure: have you looked at the existing Label Studio docs on setting this stuff up, and are we purposefully duplicating some of that here?

Reply:
I think we are repeating information elsewhere - and a future task for me would be to make sure I'm using content reuse when possible. But I'd rather have all the information up front and here where they need it rather than sending them off to other pages in different sections (in this case, the pages on importing data from cloud storage).